pkgs/development/libraries/spirv-headers/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "spirv-headers";
-  version = "1.5.3";
+  version = "unstable-2021-08-11";
```

Comment:
(makes me long for the time when NixOS/RFCs#107 is finalised, but it seems to be a package only used for one rev-dependency and almost never installed directly)